README.md

```diff
@@ -1144,6 +1146,17 @@ rake chewy:journal:apply["$(date -v-1H -u +%FT%TZ)"] # apply journaled changes f
 rake chewy:journal:apply["$(date -v-1H -u +%FT%TZ)",users] # apply journaled changes for the past hour on UsersIndex only
 ```
 
+For the cases when journal has grown up to enormous size, classical way of deletion is not quite possible. Fortunately chewy internally uses [delete-by-query](https://www.elastic.co/guide/en/elasticsearch/reference/7.17/docs-delete-by-query.html#docs-delete-by-query-task-api) ES function which support async execution with batching and [throttling](https://www.elastic.co/guide/en/elasticsearch/reference/current/docs-delete-by-query.html#docs-delete-by-query-throttle).
+
+Available options are listed below, they can by set by ENV variables:
+* `WAIT_FOR_COMPLETION` - boolean flag, options controls async execution, It waits by default, when set to `false`, Elasticsearch performs some preflight checks, launches the request, and returns a task you can use to cancel or get the status of the task
+* `REQUESTS_PER_SECOND` - float, The throttle for this request in sub-requests per second
+* `SCROLL_SIZE` - integer, Size of the scroll request that powers the operation
```

Comment:
```suggestion
* `SCROLL_SIZE` - integer. The size of the scroll request that powers the operation.
```

Reply:
I'm not quite sure what that means. :)

Reply:
I think that we could 2-3 sentences of explaination of how the delete by query works in https://github.com/toptal/chewy/pull/857/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R1149. I'd describe what it is and how is uses the bulk API. With such an introduction it might be easier to understand all the params that we expose as ENVs. WDYT?

Reply:
The way `delete-by-query` works in described in the link. If you don't understand the options of `delte-by-query` because you don't know how it works you reach ES documentation. I don't see much point of copying ES documentation here.